rundeckapp/src/test/groovy/rundeck/services/RdJobServiceSpec.groovy

```diff
@@ -182,15 +181,15 @@ class RdJobServiceSpec extends Specification implements DataTest {
         def actual = service.detectJobChanges(se, job, event)
 
         then:
-        actual.scheduledJobName == expectedJobName
+        actual.scheduledJobName == uuid
         actual.renamed == renamed
-        event.changeinfo.origName == expectedJobName
+        event.changeinfo.origName == uuid
         if(renamed) event.changeinfo.rename
```

Comment:
!!FYI this line will never cause an assertion failure, due to the way spock only modifies the immediate statements in the block into assertions, and doesn't descend into if statements. 

This will need to be changed to:
```suggestion
        if(renamed){
            assert event.changeinfo.rename
        }
```